src/Search/SearchItem.tsx

```diff
@@ -27,10 +40,38 @@ export function SearchItem({ item }: SearchItemProps) {
   }, [item.active]);
 
   return (
-    <SearchLink to={item.url} tabIndex={0} innerRef={ref} data-component-name="Search/SearchItem">
-      <Title>{item.title}</Title>
-      <Description>{highlight(item.chunks)}</Description>
-      <Path>{item.path.join(' > ')}</Path>
+    <SearchLink to={item.url} tabIndex={0} innerRef={ref}>
+      <Operaition>
+        {item.httpVerb ? <OperationBadge type={item.httpVerb}>{item.httpVerb}</OperationBadge> : null}
+        {highlight(item.pathName)}
+      </Operaition>
+      <Title>
+        {highlight(item.title)}
+      </Title>
+      <Description>
+        {highlight(item.text)}
+      </Description>
+
+      {item.parameters?.length
+        ?
+        <Parameters>
+          {item.parameters.map(param => {
+            const path = `${param.place} → ${param.path?.length ? param.path?.join(' → ') + ' → ': ''}`;
+            return <>
+              <Place>
+                {path}{highlight(param.name)} <br/>
+                {highlight(param.description)}
+              </Place>
+            </>
+            })
+          }
+          {moreItems ? <MoreText>and {moreItems} more...</MoreText> : null}
+        </Parameters>
```

Comment:
Let's extract it into separate component
```suggestion
        <Parameters parameters={item.parameters} />
```